src/gamefalloutnv.cpp

```diff
@@ -246,7 +244,11 @@ QString GameFalloutNV::steamAPPId() const
 
 QStringList GameFalloutNV::primaryPlugins() const
 {
-  return {"falloutnv.esm"};
+  return {
```

Comment:
I think we're going to remove this for now. There are both technical and legacy reasons to leave this alone and there are use cases for FNV players to allow them to toggle and reorder the DLCs if they want to. TTW is a different matter.